trace/cloud-trace-demo-app/setup.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
nit: I super dislike including bash scripts to launch demos for users. 

The point of our samples is to teach folks how to deploy the sample. Abstracting in a script makes it harder for users to understand what is happening and learn how to do it on their own. 

Additionally, we don't have any testing set up, so this bash script could break and we don't know. It's better to have these instructions exist as apart of the c.g.c docs linking to this sample so that there is one place to make sure instructions are updated.

Reply:
Todo: @ZiweiZhao Explain our situation here.

Reply:
Hi, we had several discussions around the bash scripts here. 
The purpose of our demo app is to help user get live tracing data in their project. We do not focus on the learning of how to deploy services to GKE and how to send http request among each service. Actually including extra learning here might add complexity to the demo app.

Reply:
@kurtisvg for comment

Reply:
I still disagree - the reason I dislike scripts like these it's the user doesn't know what they need to don't need to know until _after_ they already know what they need too. If the focus isn't to teach folks how to deploy stuff to GKE then why include it in the sample at all? Just have it be a flask app and leave the irrelevant parts out. 

That said we don't have any guidance or policy around this - so still a nit, and still up to you folks to decide what you prefer. Just keep in mind that it's untested and we won't know if it breaks unless someone complains.